index.php

```diff
@@ -0,0 +1,127 @@
+<?php
+?>
+
+<html lang="en-GB">
+
+	<head>
+		<title> Welcome!</title>
+		<link rel="stylesheet" type="text/css" href="style.css">
+	</head>
+
+<body>
+<nav class="nav_bar">
+    <p class="welcome">Welcome to my own personal Plant Database</p>
+    <div class="links_nav">
+        <a href="form.php">Add new plants to the database</a>
+    </div>
+</nav>
+
+<main>
+    <h1>Hi!</h1>
+    <h1>Here you can see my stunning house plant collection:</h1>
+
+    <div class="plants_collection">
+        <div class="plants">
+            <img class="prayer_plant" src="images/prayer_plant.jpg" alt="here you can see my stunning Prayer Plant or Calathea medallion">
+            <p>Prayer Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Calathea medallion</ul>
```

Comment:
should `<ul>` tags not be `<li>`'s in a `<ul>` instead -- so that each item is a list item instead of each item being an unordered list of it's own?

Reply:
I took the ul tags away.